README.md

```diff
@@ -97,29 +97,49 @@ tokio-stream = { version = "0.1" }
 near-lake-framework = "0.6.1"
 ```
 
-## Cost estimates
+## Cost estimates (Updated Mar 10, 2022 with more precise calculations)
 
-**TL;DR** approximately $18.15 per month (for AWS S3 access, paid directly to AWS) for the reading of fresh blocks
+**TL;DR** approximately $20 per month (for AWS S3 access, paid directly to AWS) for the reading of fresh blocks
+
+### Historical indexing
+
+| Blocks | GET | LIST | Subtotal GET | Subtotal LIST | Total $ |
+|---|---|---|---|---|---|
+| 1000 | 5000 | 4 | 0.00215 | 0.0000216 | $0.00 |
```

Comment:
In the next iteration (in the future parallel universe) we might unpack why "historical indexing" uses 4 LIST requests while "tip of the network indexing" uses 1000.